app/controllers/v0/sessions_controller.rb

```diff
@@ -42,6 +42,16 @@ def saml_callback
       end
     end
 
+    def renew
+      if @session.save
+        @current_user.save
+        # TODO: - save other stuff associated with a "vets.gov session"
```

Comment:
According to `redis.yml` we have the following models using Redis:

| Model   | Action Required                     |
|---------|-------------------------------------|
| Session | Accounted for                       |
| User    | Accounted for                       |
| RX      | ???                                 |
| SM      | ???                                 |
| MVI     | We should re-save if it's available |

I will add logic to re-save `MVI` in the event that we have it.  I'm not too familiar with RX and SM and will need input on how to extend these, or if it's even desirable to extend them.